v2/invoke_test.go

```diff
@@ -68,6 +72,47 @@ func TestInvokeSuccess(t *testing.T) {
 		t.Errorf("slept %d times, should not have slept since the call succeeded", int(sp))
 	}
 }
+func TestInvokeAPIError(t *testing.T) {
```

Comment:
When printed out, the apierr and err appear to be the same but test still fails @noahdietz